src/server/req_delete.c

```diff
@@ -1270,7 +1285,7 @@ post_delete_mom1(struct work_task *pwt)
 				return;
 			}
 			/* 2nd try, use SIGTERM */
-			rc = issue_signal(pjob, sigt, post_delete_mom1, preq_clt, NULL);
+			rc = issue_signal(pjob, sigt, post_delete_mom1, preq_clt);
```

Comment:
Don't we need to reply to preempt req on line 1307 as well? when mom rejects the request stating the job was not found.